content/html/concepts/elements/terms/label/label.md

```diff
@@ -0,0 +1,111 @@
+---
+Title: '<label>'
+Description: 'Identifies captions for HTML Form elements such as <input> and other UI elements.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Elements'
+  - 'Form'
+  - 'Label'
+  - 'Input'
+  - 'Interface'
+  - 'Tags'
+  - 'UI'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<label/>`** element is used to identify captions for other HTML elements. `<label/>` is most commonly used with form UI elements such as the [`<input>`](https://www.codecademy.com/resources/docs/html/elements/input/input.md) element. The `<label>` tag also supports the *Event Attributes* and the *Global Attributes* in HTML.
+
+## Syntax
+
+```html
+<label>Label display text</label>
```

Comment:
```suggestion
<label for="">
  <!-- Labeled content goes here -->
</label>

<!-- Labeled elements, not contained -->
```

Reply:
i wanted to save the "labeled elements ..." for the usage section? thoughts?